requests/sessions.py

```diff
@@ -98,6 +99,10 @@ def resolve_redirects(self, response, stream=False, timeout=None,
         request = response.request
 
         while response.is_redirect:
+            if len(response.raw.headers.getlist('location')) > 1:
```

Comment:
This will need to include a `getattr` check. Frankly, it's a bit implementation detaily, so we may want to hide this in a utility function.